src/actions/const.js

```diff
@@ -2,3 +2,4 @@
 /* Populated by react-webpack-redux:action */
 export const CHANGE_COLOR = 'CHANGE_COLOR';
 export const CHANGE_WIDTH = 'CHANGE_WIDTH';
+export const FINISH_PEN_PREVIEW = 'FINISH_PEN_PREVIEW';
```

Comment:
@dadaa 
This action is need to hide the preview.
Is there a better timing to hide that?